pkg/native/unserializers/unserializer_cdx15.go

```diff
@@ -0,0 +1,115 @@
+package unserializers
+
+import (
+	"fmt"
+	"io"
+
+	cdx "github.com/CycloneDX/cyclonedx-go"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/bom-squad/protobom/pkg/reader/options"
+	"github.com/bom-squad/protobom/pkg/sbom"
+)
+
+type UnserializerCDX15 struct {
+	UnserializerCDX14
+}
+
+// ParseStream reads a CycloneDX 1.5 from stream r usinbg the offcial CycloneDX
+// libraries and returns a protobom document with its data.
+func (u *UnserializerCDX15) ParseStream(_ *options.Options, r io.Reader) (*sbom.Document, error) {
+	bom := new(cdx.BOM)
+	decoder := cdx.NewBOMDecoder(r, cdx.BOMFileFormatJSON)
+	if err := decoder.Decode(bom); err != nil {
+		return nil, fmt.Errorf("decoding cyclonedx: %w", err)
+	}
+
+	md := &sbom.Metadata{
+		Id:      bom.SerialNumber,
+		Version: fmt.Sprintf("%d", bom.Version),
+		// Name:    ,
+		Date:          &timestamppb.Timestamp{},
```

Comment:
nit: Capture the date here (strongly suggested) or leave it as nil (preferably not) otherwise it will revert to the unix epoch

Reply:
Thanks!

Reply:
Should I do it for CDX 1.4 as well?

Reply:
Cool, but we can do it in a follow up too.